### ISSUE TITLE
Feature request: --show-zmws 

### ISSUE THREAD

**Issue by mrvollger** [PacificBiosciences/zmwfilter#2]
Hello!

I really like `zmwfilter` and I intended to switch to it entirely over `bamseive` but it doesn't have a command that extracts all the zmws into plain text like `--show-zmws` in `bamseive`. This is too bad because zmwfilter is much faster and is much easier to install, but without `--show-zmws` I am stuck using bamseive. 

Any chance this could be added as a feature?

(Tagging @ctsa)

Thanks in advance!
Mitchell

**Comment by armintoepfer** [comment]
Could you remind me what `--show-zmws` is doing? What's your use case?

**Comment by mrvollger** [comment]
Thanks @armintoepfer for the response! 

`bamseive --show-zmws` just makes a plain text file with one zmw hole number per line, e.g.:
```
$ bamsieve --show-zmws /mmfs1/home/mvollger/stergachislab/data/PS00072_WGA_K562_m6A_Mtase_treated/PACBIO_DATA/XASHU_20201216_S64018_PL100163568-1_C01_bc1019-bc1019.bam | head 
3
4
21
28
56
58
171
184
198
231
```

I have made a tool for m6a predictions using hifi bams and I need to be able to extract zmws so I can split the bam and distribute over multiple separate jobs on our cluster. The steps are roughly: I make a text file all the with zmws/holes using `bamseive` and `--show-zmws` and then I chunk that into ~100 pieces and use zmwfilter with —include on each chunk to make ~100 smaller bams which can the be processed independently and merged. 

Basically it is a way to recreate the  `--chunks 1/n` option that exists in many PacBio tools. But my program is not in cpp so I cannot use the pbbam api, so I need to chunk the bam on the command line first. I think this feature would have use cases outside of my own, since this idiom of dividing a bam file seems like it could be a common task. 

Thanks again,
Mitchell

**Comment by armintoepfer** [comment]
Can you use the pbindexdump workaround for now that I've created for the same tweet thread? 
https://gist.github.com/armintoepfer/07ed52dfe3520eab2086ac117552e039

**Comment by mrvollger** [comment]
Yeah that works great for me as a workaround and Zev just suggested pbindex dump to me as well, so I have been using that in the interim. However, I hope there is still interest in implementing this feature since I think it will come up fairly often?

**Comment by armintoepfer** [comment]
Yep, but not until tomorrow, this the faster workaround for now

**Comment by armintoepfer** [comment]
We have an update to pbindexdump and zmwfilter. This will be made available early next year via https://github.com/PacificBiosciences/pbtk. This repo will be archived soon, fyi.

**Comment by mrvollger** [comment]
Great! Looking forward to trying it out soon!

**Comment by mrvollger** [comment]
Any updates on a new release for pbtk?

**Comment by armintoepfer** [comment]
I'll try to target a release for next week

**Comment by armintoepfer** [comment]
We have a new meta repo that combines all one-off tools https://github.com/PacificBiosciences/pbtk. This repo and conda recipe will become a simple dependency of `pbtk`. The latest zmwfilter binary should be in there by now. I'm in the process of removing the binary from this repo. Feel free to remove zmwfilter package and install `pbtk` instead. Or just get the binary from the github pbtk tarball ;)

**Comment by armintoepfer** [comment]
I'm also closing the issues tab, everything shall go into https://github.com/PacificBiosciences/pbbioconda
